### PR TITLE
Minor changes after merge conflicts

### LINKED ISSUE #733: Issues remaining after #732
Because of the decision to do many PRs that conflated revers and updates, it was very difficult to find issues. Here's what I found so far

1. Merge conflict left sitting here (should be handled by CI) https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/catalog-v001.xml#L22
2. Non HTTPS orcid url (should be handled by CI)
https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6485
3. Five spurious `^^xsd:dateTime` added (these are actually correct, but not it's frustrating that they just showed up during the recent mess)
   - https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6660
   - https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6673
   - https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6686
   - https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6699
   - https://github.com/oborel/obo-relations/blob/97a906a4d3d6ac2b44339f1928a0d2555523dc49/src/ontology/ro-edit.owl#L6712

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #733

### CODE REVIEW

**Review by matentzn — APPROVED** [review]

**Comment by wdduncan** [comment]
Is this ready to merge?

**Comment by anitacaron** [comment]
Yes, @wdduncan. I am waiting for the second approval.

**Comment by wdduncan** [comment]
@cthoyt Do you have time to review?

**Review by cthoyt — APPROVED** [review]

**Review by wdduncan — APPROVED** [review]
A number of the changes are hard to follow due spurious diffs (i.e., a line being deleted and inserted).   
I'll have to trust that everything is fine :)

**Comment by anitacaron** [comment]
Thanks. This is simply reordering the file.
